clang-tools-extra/clang-tidy/bugprone/SuspiciousIncludeCheck.h

```diff
@@ -28,9 +27,12 @@ class SuspiciousIncludeCheck : public ClangTidyCheck {
   SuspiciousIncludeCheck(StringRef Name, ClangTidyContext *Context);
   void registerPPCallbacks(const SourceManager &SM, Preprocessor *PP,
                            Preprocessor *ModuleExpanderPP) override;
+  void storeOptions(ClangTidyOptions::OptionMap &Opts) override;
 
   FileExtensionsSet HeaderFileExtensions;
   FileExtensionsSet ImplementationFileExtensions;
+  std::optional<StringRef> IgnoredRegexString;
```

Comment:
Please make all data members private.

Reply:
You said that before, I answered with a question, and got no answer. @vbvictor came with an answer and I went with that.

So if you can't react on my questions/answers, I'll certainly will not do anything.